libs/react-helpers/src/lib/format/number.ts

```diff
@@ -2,11 +2,15 @@ import { BigNumber } from 'bignumber.js';
 import memoize from 'lodash/memoize';
 import { getUserLocale } from './utils';
 
-export function addDecimal(value: string, decimals: number): string {
+export function addDecimal(
+  value: string,
+  decimals: number,
+  decimalPrecision?: number
```

Comment:
I forget the synax for default with TS so double check this but: suggest defaulting here

```suggestion
  decimalPrecision = decimals: number
```